src/appsettings.dncengpublic-prod.json

```diff
@@ -26,7 +26,8 @@
     "BuildPool.Windows.Amd64.VS2019.Pre.ES.Open",
     "BuildPool.Windows.10.Amd64.VS2019.BT.Open",
     "BuildPool.Windows.10.Wasm.Open",
-    "BuildPool.Windows.10.Amd64.VS2019.tmp.Open"
+    "BuildPool.Windows.10.Amd64.VS2019.tmp.Open",
```

Comment:
If this one's deprecated, do we want to delete it?

Reply:
If I understood correctly, we deprecated that name and are going to call it `BuildPool.Windows.VS2019.Scouting.Open`?